.overcommit.yml

```diff
@@ -9,6 +9,8 @@ PreCommit:
 
   HardTabs:
     enabled: true
+    exclude:
+      - 'lib/haml_lint/linter/README.md'
```

Comment:
I added this because there are examples that use hard tabs to show what not to do. I tried to figure out a way to only ignore those lines but wasn't able to find anything. If there's a way to disable overcommit's check for only specific lines, I'll be happy to make that change.